originals/t.yaml

```diff
@@ -76,13 +76,26 @@
   meta:
     genre: [Action, Adventure]
 
-- name: The Settlers II
+- name: [The Settlers, The Settlers (video game)]
+  names: [Die Siedler II, Serf City]
+  meta:
+    genre: [RTS]
+    subgenre: [City Building]
+    theme: [Medieval]
+
+- name: [The Settlers II, The Settlers II]
+  names: [Die Siedler II]
   meta:
     genre: [RTS]
+    subgenre: [City Building]
+    theme: [Medieval, Alternate Historical, Fantasy]
 
-- name: The Settlers III
+- name: [The Settlers III, The Settlers III]
+  names: [Die Siedler III]
   meta:
     genre: [RTS]
+    subgenre: [City Building]
+    theme: [Medieval, Egyptian, Alternate Historical, Fantasy]
```

Comment:
I don't think `Medieval` is right here, from the description it seems more like mythological, and for time period antiquity or ancient history. Unfortunately the Giant Bomb themes aren't the best. It would be good to come up with a better set of themes later.

Reply:
Shall we create Medieval? For now?

Reply:
I would have used Ancient if that was a theme, unfortunately it isn't :/